src/mame/drivers/sms_bootleg.cpp

```diff
@@ -272,34 +435,169 @@ DRIVER_INIT_MEMBER(smsbootleg_state,sms_supergame)
 	{
 		rom[i] ^= 0x80;
 	}
+
+	for (int i = 0;i < 0x200;i++)
+	{
+		m_unpaged->configure_entry(i, &m_mainrom[(i & 0x1fe) * 0x4000]);
+		m_page0->configure_entry(i, &m_mainrom[0x400 + i * 0x4000]);
+		m_page1->configure_entry(i, &m_mainrom[i * 0x4000]);
+		m_page2->configure_entry(i, &m_mainrom[i * 0x4000]);
```

Comment:
I dislike how you're micromanaging the implementation here.  I'm simply closing this, somebody else can do it.